codewind-che-sidecar/tests/cronjob-sidecar.sh

```diff
@@ -59,7 +59,7 @@ else
   exit 1
 fi
 
-export CHE_INGRESS_DOMAIN=che-$NAMESPACE.$CLUSTER_IP.nip.io
+export CHE_INGRESS_DOMAIN=$(kubectl get routes --selector=component=che -o jsonpath="{.items[0].spec.host}" 2>&1)
```

Comment:
I know we're running `oc project` before this command and that the cronjob will only run on OCP and OKD for the near future, but do we want to add the `-n namespace` are to the command here too? In case we ever need to run the cronjob on other Kubes.

It's up to you, I'm fine merging this either way.